### PR TITLE
Cleanup OAS definitions(Change product API to APK, remove scopes and other unused stuff)

### PR DESCRIPTION
**Pull request by sanjeewa-malalgoda**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml [hunk 1]
```diff
@@ -5547,7 +5501,7 @@ components:
       name: X-WSO2-Tenant
       in: header
       description: |
-        For cross-tenant invocations, this is used to specify the tenant domain, where the resource need to be
+        For cross-tenant invocations, this is used to specify the /organization domain, where the resource need to be
```
Comment (Mariangela) [inline devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml:5504]: @sanjeewa-malalgoda , looks like we do not need `/` before the word "organization".

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml [hunk 2]
```diff
@@ -14,17 +14,16 @@
...
-    Please see [full OpenAPI Specification](https://raw.githubusercontent.com/wso2/carbon-apimgt/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml) of the API which is written using [OAS 3.0](http://swagger.io/) specification.
+    This document specifies a **RESTful API** for WSO2 **API Platform for Kubernetes(APK)** - **Developer Portal**.
 
     # Authentication
-    The Developer Portal REST API is protected using OAuth2 and access control is achieved through scopes. Before you start invoking
-    the API, you need to obtain an access token with the required scopes. This guide will walk you through the steps
-    that you will need to follow to obtain an access token.
-    First you need to obtain the consumer key/secret key pair by calling the dynamic client registration (DCR) endpoint. You can add your preferred grant types
-    in the payload. A Sample payload is shown below.
+    The Developer Portal REST API is protected using OAuth2 and access control is achieved through scopes(in the intitial release
```
Comment (Mariangela) [inline devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml:22]:
```suggestion
    The Developer Portal REST API is protected using OAuth2 and access control is achieved through scopes (in the initial release 
```

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml [hunk 3]
```diff
@@ -14,17 +14,16 @@
...
+    This document specifies a **RESTful API** for WSO2 **API Platform for Kubernetes(APK)** - **Developer Portal**.
 
     # Authentication
-    The Developer Portal REST API is protected using OAuth2 and access control is achieved through scopes. Before you start invoking
-    the API, you need to obtain an access token with the required scopes. This guide will walk you through the steps
-    that you will need to follow to obtain an access token.
-    First you need to obtain the consumer key/secret key pair by calling the dynamic client registration (DCR) endpoint. You can add your preferred grant types
-    in the payload. A Sample payload is shown below.
+    The Developer Portal REST API is protected using OAuth2 and access control is achieved through scopes(in the intitial release 
+    we wont have scope support for APIs). Before you start invoking the API, you need to obtain an access token with the required scopes.
```
Comment (Mariangela) [inline devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml:23]:
```suggestion
    WSO2 does not provide scope support for APIs). Before you start invoking the API, you need to obtain an access token with the required scopes. 
```

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml [hunk 4]
```diff
@@ -2909,7 +2891,7 @@ paths:
       summary: |
         Get a List of Supported SDK Languages
       description: |
-        This operation will provide a list of programming languages that are supported by the swagger codegen library for generating System Development Kits (SDKs) for APIs available in the API Manager Developer Portal
+        This operation will provide a list of programming languages that are supported by the swagger codegen library for generating System Development Kits (SDKs) for APIs available in the API Platform for Kubernetes(APK) Developer Portal
```
Comment (Mariangela) [inline devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml:2894]:
```suggestion
        This operation will provide a list of programming languages that are supported by the swagger codegen library for generating System Development Kits (SDKs) for APIs available in the API Platform for Kubernetes (APK) Developer Portal.
```

**Review by Mariangela — APPROVED** [review]
@sanjeewa-malalgoda , I added some comments.

File: backoffice/backoffice-domain-service/ballerina/modules/backoffice/resources/backoffice-api.yaml [hunk 5]
```diff
@@ -103,7 +103,6 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_view
```
Comment (Mariangela) [inline backoffice/backoffice-domain-service/ballerina/modules/backoffice/resources/backoffice-api.yaml:105]: Don't we need to change `apim:` in this instance and the rest?

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml [hunk 6]
```diff
@@ -92,11 +91,11 @@ info:
 
     # Try out in Postman
     If you want to try-out the embedded postman collection with "Run in Postman" option, please follow the guidelines listed below.
-    * All of the OAuth2 secured endpoints have been configured with an Authorization Bearer header with a parameterized access token. Before invoking any REST API resource make sure you run the `Register DCR Application` and `Generate Access Token` requests to fetch an access token with all required scopes.
-    * Make sure you have an API Manager instance up and running.
+    * All of the OAuth2 secured endpoints have been configured with an Authorization Bearer header with a parameterized access token. 
+    * Before invoking any REST API resource make sure you run the `Register DCR Application` and `Generate Access Token` requests to fetch an access token with all required scopes.
+    * Make sure you have an API Platform for Kubernetes(APK) instance up and running.
     * Update the `basepath` parameter to match the hostname and port of the APIM instance.
```
Comment (Mariangela) [inline devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml:97]:
```suggestion
    * Update the `basepath` parameter to match the hostname and port of the APK instance.
```

File: devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml (hunk 1 above)
Comment (sanjeewa-malalgoda) [reply devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml]: fixed

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
- fixes https://github.com/wso2/apk/issues/813
- Partly fixes https://github.com/wso2/apk/issues/814 This fix is only applicable for the following YAMLs.
         - backoffice-api.yaml
         - devportal-api.yaml

**Pull request merged**